src/doc/DocAnnotator.js

```diff
@@ -240,26 +242,21 @@ class DocAnnotator extends Annotator {
         return location;
     };
 
-    /**
-     * Override to factor in highlight types being filtered out, if disabled. Also scales annotation canvases.
-     *
-     * @param {number} pageNum Page number
-     * @return {void}
-     */
-    renderPage(pageNum: number) {
+    /** @inheritdoc */
+    resetPopoverUI(pageNum?: number) {
```

Comment:
It seems like this is doing a lot more than changing popovers. It looks to be removing ranges, highlights, and scaling canvases. Should this be renamed?

Reply:
Renamed it to resetAnnotationUI since the method cleans up any UI related to the current pending annotation